specification/search/data-plane/Azure.Search/preview/2024-05-01-Preview/searchindex.json

```diff
@@ -1320,7 +1326,14 @@
         "semantic": {
           "$ref": "#/definitions/SemanticDebugInfo",
           "readOnly": true,
+          "x-nullable": true,
```

Comment:
Why is this marked nullable? That means the user can pass `null` as a value and is probably not what you mean. This is only useful for json+mergepatch.